### DISCUSSION TITLE
Points in 3d

### DISCUSSION THREAD

**Discussion by labriki** [gempy-project/gempy#856] [Q&A]
I am a beginner in Gempy. I installed gempy according to the installation guidelines in an environment. I launched the "Get Started" tutorial, everything goes well except at 'gp.plot_3d(geo_model)' where the 3d figure appears but without the points. Whereas in '2d' it works. What is the solution?
Gempy ver. = 2.3.1, Python ver.=3.8.5

**Comment by javoha** [comment]
Hi,
just to make sure: The first 3D Plot in this example is supposed to be "empty", as no points were added to this point. I suppose it doesnt work for you during the later stages?

**Reply by labriki** [reply]
exactly this doesn't work in any of the tutorials I've experienced. but the directions (small arrows) are displayed
